uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+require_once 'hrleaveandabsences.civix.php';
+
+/**
+ * Implements hook_civicrm_config().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_config
+ */
+function hrleaveandabsences_civicrm_config(&$config) {
+  _hrleaveandabsences_civix_civicrm_config($config);
+}
+
+/**
+ * Implements hook_civicrm_xmlMenu().
+ *
+ * @param array $files
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_xmlMenu
+ */
+function hrleaveandabsences_civicrm_xmlMenu(&$files) {
+  _hrleaveandabsences_civix_civicrm_xmlMenu($files);
+}
+
+/**
+ * Implements hook_civicrm_install().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_install
+ */
+function hrleaveandabsences_civicrm_install() {
+  $reportWeight = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'Reports', 'weight', 'name');
```

Comment:
I was using the hrabsences extension as a reference to implement this and it uses Reports.  It made sense to me to use it here too, since it is the first non-civihr extension on the menu (not counting search and contacts) its weight - 1 works as a way to "add this menu item after the last civihr item"